bool hopperMaterialLevel
bool hooperBreaker
bool hooperBreakerError

